### ISSUE TITLE
Fix flaky tests

### ISSUE THREAD

**Issue by jayanthdeejay** [uclibs/ucrate#213]
### Descriptive summary

Tests are failing after merging PR #178 

### Steps to reproduce the behavior

1. Tests are failing for seeds 42315, 55877 etc...
1. bundle exec rspec  --seed 42315

### Related work

#148

**Comment by jayanthdeejay** [comment]
After running test suite multiple times, while reviewing the PR #214, found more order dependent test failures apart from the original
```ruby
rspec './spec/features/uc_shibboleth_spec.rb[1:3:1,1:4:2]' './spec/features/welcome_mailer_spec.rb[1:3:1]' --seed 42315
```
New failing examples with order-dependency are as follows:
```ruby
rspec './spec/features/batch_edit_spec.rb[1:1:2]' './spec/features/ownership_transfer_spec.rb[1:1:1:1,1:1:2:2:1]' './spec/features/sitemap_spec.rb[1:1:1]' './spec/features/welcome_mailer_spec.rb[1:1]' --seed 55682
```
Currently working on fixing the same.

**Comment by hortongn** [comment]
Closing this until we see more flakiness show up.  May have been fixed by #241
